solutions/usaco-966.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: usaco-966
+source: USACO Silver 2019 December
+title: MooBuzz
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://www.usaco.org/r/data/sol_moobuzz_silver_dec19.html)
+
+For every $15$ numbers, there are only $8$ numbers where the cows moo. (draw it out!)
+
+![](/solutions/USACO-dec19-moobuzz-silver/first30_relation.png)
+
+Therefore, we can just get the distance away $N$ is from the first $8$ by calculating $\lfloor N / 8 \rfloor \times 15$, since we have to add $15$ for every grouping of $15$. 
```

Comment:
```suggestion
Therefore, we can just get the distance away $N$ is from the first $8$ by calculating  $\lfloor \frac{N}{8} \rfloor \times 15$, since we have to add $15$ for every grouping of $15$. 
```